.travis.yml

```diff
@@ -18,4 +18,4 @@ jobs:
           sudo: required
           addons:
               chrome: beta
-          script: travis_retry yarn run functional-tests-ci
+          script: travis_retry yarn run functional-tests-ci; yarn test:e2e
```

Comment:
I think we'll want to use `&&` rather than `;` here. The latter causes the next command to execute regardless of the exit code from the prior command.